commit 5efdb4be598fc2af6937c3387586635ddf6fd2c8
Author: Alexander Graf <agraf@suse.de>
Date:   Wed Apr 17 00:37:57 2013 +0200

    KVM: PPC: MPIC: Add support for KVM_IRQ_LINE
    
    Now that all pieces are in place for reusing generic irq infrastructure,
    we can copy x86's implementation of KVM_IRQ_LINE irq injection and simply
    reuse it for PPC, as it will work there just as well.
    
    Signed-off-by: Alexander Graf <agraf@suse.de>

diff --git a/arch/powerpc/include/uapi/asm/kvm.h b/arch/powerpc/include/uapi/asm/kvm.h
index ca871067a69b..03c7819a44a3 100644
--- a/arch/powerpc/include/uapi/asm/kvm.h
+++ b/arch/powerpc/include/uapi/asm/kvm.h
@@ -26,6 +26,7 @@
 #define __KVM_HAVE_SPAPR_TCE
 #define __KVM_HAVE_PPC_SMT
 #define __KVM_HAVE_IRQCHIP
+#define __KVM_HAVE_IRQ_LINE
 
 struct kvm_regs {
 	__u64 pc;